packages/cli/src/gemini.tsx

```diff
@@ -257,12 +258,14 @@ export async function main() {
     setWindowTitle(basename(workspaceRoot), settings);
     const instance = render(
       <React.StrictMode>
-        <AppWrapper
-          config={config}
-          settings={settings}
-          startupWarnings={startupWarnings}
-          version={version}
-        />
+        <SettingsContext.Provider value={settings}>
+          <AppWrapper
+            config={config}
+            settings={settings}
```

Comment:
nit: move this into AppWrapper or modify AppWrapper to load settings from settingsContext. 
App.tsx is already adding a couple other context variables so would suggest that as the natural place.